package.json

```diff
@@ -43,14 +44,21 @@
     "deepmerge": "^4.2.1",
     "del": "^5.1.0",
     "font-awesome": "^4.7.0",
+    "headroom.js": "^0.11.0",
```

Comment:
Is this actually being used or has it been copypasta'd from someplace else. I know it's in Curzon but